sdk/testing/build.gradle.kts

```diff
@@ -9,6 +11,7 @@ otelJava.moduleName.set("io.opentelemetry.sdk.testing")
 dependencies {
   api(project(":api:all"))
   api(project(":sdk:all"))
+  api(project(":sdk-extensions:logging"))
```

Comment:
This package doesn't seem to be related to the PR, can we revert it?

Reply:
@ran8080 this line should be deleted, yes?